components/dashboard/src/admin/WorkspaceDetail.tsx

```diff
@@ -15,13 +16,17 @@ import { getProjectPath } from "../workspaces/WorkspaceEntry";
 import { WorkspaceStatusIndicator } from "../workspaces/WorkspaceStatusIndicator";
 import Property from "./Property";
 
+dayjs.extend(duration);
```

Comment:
Could we move this to somewhere like dashboard/index.ts or similar to avoid it being in a particular component? Makes it easier to track down when it's done in the app entrypoint.

Reply:
What's the problem with having that here? I think it's better to have dependencies declared in the context they are used rather than globally.

Reply:
Sure, but this is not a dependency import. It's a runtime "patch" of the object and generally if you're patching stuff, keeping that in the entrypoint means 1. you ensure it always happens before the rest of the app, 2. makes it easier to find.

For example, if another component tried to use this, and it got loaded first, it wouldn't have the "duration" extension loaded.

Reply:
Sorry, decided to remove the comment as I didn't feel it's worth arguing :-)

Reply:
`index.tsx` already does it, for other ones too

```
    // Configure libraries
    dayjs.extend(relativeTime);
    dayjs.extend(utc);
```

Reply:
Uhm ... I now realized there is actually a place where we setup this lib. So moved it there. 😇